index.js

```diff
@@ -313,6 +313,65 @@ MochaJUnitReporter.prototype.getTestsuiteData = function(suite) {
   return testSuite;
 };
 
+/**
+ * Given a fully qualified test name and test case context, derive a list of user-defined
+ * tags, if present.
+ *
+ * @param {String} name - the fully qualified name of the test
+ * @param {object} test - test case
+ */
+function getTagsFromTestcaseData(name, test) {
+  // extract tags and teams from the test name
+  // this regex matches any whole word beginning with # or @
+  // includes words surrounded by parenthesis, square brackets, and punctuation
+  var tagsFromName = name.match(/([#|@]\w+)/g) || [];
```

Comment:
would something like `manager-tools` or `developer-experience` be matched as a whole word in this context?

Reply:
nice catch! i've updated it to allow `-` and `_`